libs/core/src/lib/tabs/tab-nav/tab-nav.component.scss

```diff
@@ -1 +1,18 @@
 @import '~fundamental-styles/dist/tabs';
+.fd-tabs__link.is-selected {
+    .fd-tabs__tag:after {
+        background-color: #0854a0;
+        background-color: var(--fdTabs_Selected_Color, var(--sapSelectedColor, #0854a0));
+    }
+    .fd-tabs__icon {
+        color: #fff;
+        color: var(--fdTabs_Selected_Icon_Color, var(--sapContent_ContrastIconColor, #fff));
+        background-color: #0854a0;
+        background-color: var(--fdTabs_Selected_Color, var(--sapSelectedColor, #0854a0));
+
+        &::after {
+            background-color: #0854a0;
+            background-color: var(--fdTabs_Selected_Color, var(--sapSelectedColor, #0854a0));
+        }
+    }
+}
```

Comment:
This needs to go in Fund-styles. We are no longer allowed to approve PRs with css that belongs in fund-styles.